### PR TITLE
Use <video> element for video attachments

### PR DESCRIPTION
**Pull request by ralphtheninja**
Closes https://github.com/deltachat/deltachat-desktop/issues/112

@#$%^@ the `` tag, lets just use `<video>` and be done with it?

Result:

screenshot from 2018-10-31 22-58-34

### CODE REVIEW

**Comment by ralphtheninja** [comment]
Might need some css love though because the "lock" and timestamp are slightly hidden underneath.

**Comment by ralphtheninja** [comment]
@Jikstra Probably faster if you add a commit with appropriate css changes on this branch.

**Comment by ralphtheninja** [comment]
screenshot from 2018-11-02 10-16-16

File: src/renderer/components/ChatView.js [hunk 1]
```diff
@@ -204,12 +204,16 @@ class RenderMessage extends React.Component {
...
+        url: msg.file,
+        contentType: convertContentType(message.filemime),
+        filename: msg.text
+      }
     } else {
       props.text = msg.text
     }
 
-    return (<Message {...props} />)
+    return (<div className='MessageWrapper'><Message {...props} /></div>)
```
Comment (karissa) [inline src/renderer/components/ChatView.js]: this would also be a good case for `styled-components`

Comment (ralphtheninja) [reply src/renderer/components/ChatView.js]: TIL

**Comment by karissa** [comment]
lgtm, let's refactor to use styled-components (or something similar) soon though otherwise the css file will start getting really frustratingly long

**Pull request merged**